### ISSUE TITLE
Roomba i7+ does not update state in the HomeBridge Accessories page.

### ISSUE THREAD

**Issue by shiddy** [karlvr/homebridge-roomba2#15]
## Current Behavior

The Roomba i7+ intermittently shows "Not Responding" when the "Auto Refresh Enabled" setting is confuigred within the config. Initially I had some success in activating the i7 from the Home app, however after it docked and emptied it's bin it never updated the state within the Home app nor the HomeBridge Accessories page regardless of how much time elapsed.

Restarting the HomeBridge server will correctly provide the state of the machines, but after any run of the machines the state remains as though the roomba is running.

## Steps to Reproduce (for bugs)
Use the Configuration provided below and run the roomba and then stop the roomba via either the Homebridge Accessories page or the Home app.

## Versions
Homebridge: v1.1.6
Node: v14.15.0
Plugin: v1.1.0

## Configuration
```json
        {
            "name": "Rosie",
            "model": "i7+",
            "blid": "...",
            "robotpwd": "...",
            "ipaddress": "192.X.X.X",
            "autoRefreshEnabled": true,
            "dockContactSensor": true,
            "binContactSensor": true,
            "accessory": "Roomba2"
        },
```

## Homebridge Log / Command Output
```
[23/11/2020, 07:02:23] [Rosie] Initializing Roomba2 accessory...
[23/11/2020, 07:02:23] [Rosie] Enabling autoRefresh every 5 seconds
...
[23/11/2020, 07:04:03] [Rosie] Running status requested
[23/11/2020, 07:04:03] [Rosie] Docked status requested
[23/11/2020, 07:04:03] [Rosie] Bin status requested
[23/11/2020, 07:04:03] [Rosie] Battery status requested
...
[23/11/2020, 07:04:23] [Rosie] Starting Roomba
[23/11/2020, 07:04:25] [Rosie] Connected to Roomba
[23/11/2020, 07:04:25] [Rosie] Roomba is running
...
[23/11/2020, 07:17:00] [Rosie] Running status requested
[23/11/2020, 07:17:00] [Rosie] Docked status requested
[23/11/2020, 07:17:00] [Rosie] Bin status requested
[23/11/2020, 07:17:00] [Rosie] Battery status requested
[23/11/2020, 07:17:03] [Rosie] Roomba pause and dock
[23/11/2020, 07:17:05] [Rosie] Connected to Roomba
[23/11/2020, 07:17:05] [Rosie] Roomba is pausing
[23/11/2020, 07:17:05] [Rosie] Roomba paused, returning to Dock
[23/11/2020, 07:17:05] [Rosie] Roomba is still running. Will check again in 3 seconds
[23/11/2020, 07:17:06] [Rosie] Roomba is still running. Will check again in 3 seconds
[23/11/2020, 07:17:06] [Rosie] Roomba is still running. Will check again in 3 seconds
[23/11/2020, 07:17:06] [Rosie] Roomba is still running. Will check again in 3 seconds
[23/11/2020, 07:17:06] [Rosie] Roomba is still running. Will check again in 3 seconds
[23/11/2020, 07:17:06] [Rosie] Roomba is still running. Will check again in 3 seconds
[23/11/2020, 07:17:06] [Rosie] Roomba has stopped, issuing dock request
[23/11/2020, 07:17:06] [Rosie] Roomba docking
...
[23/11/2020, 07:20:22] [Rosie] Running status requested
[23/11/2020, 07:20:22] [Rosie] Battery level requested
[23/11/2020, 07:20:22] [Rosie] Charging status requested
[23/11/2020, 07:20:22] [Rosie] Battery status requested
[23/11/2020, 07:20:22] [Rosie] Docked status requested
[23/11/2020, 07:20:22] [Rosie] Bin status requested
```
 no other changes in the logs other than repeated requests for state.

**Comment by dvcrn** [comment]
Probably duplicate of https://github.com/iRayanKhan/homebridge-roomba2/issues/10

**Comment by iRayanKhan** [comment]
Do you have your CacheTTl defined?

**Comment by khad** [comment]
@shiddy I had the same (or at least a very similar) problem. I am also using an i7+. I had assumed it was not specific to that model, but perhaps it was.

I'm testing the latest beta at the moment and will follow up here to let you know if it seems to have resolved things. If you want to give it a try, you can choose 🔧 > "Install Previous Version" and select the beta.

<img width="708" alt="Screen Shot 2021-09-22 at 9 30 53 PM" src="https://user-images.githubusercontent.com/758113/134454835-d8ecb240-ec5f-454c-bc6d-177e4e308d24.png">

**Comment by khad** [comment]
So far this appears to be fixed in 1.3.0-beta.6: https://github.com/karlvr/homebridge-roomba2/issues/75#issuecomment-926956051

@karlvr I think this issue can also be closed now that #75 is closed.
